src/tree_schema.h

```diff
@@ -250,38 +250,85 @@ struct lyxp_expr;
 
 #define LYS_NODETYPE_MASK 0xffff  /**< Mask for nodetypes, the value is limited for 16 bits */
 
+/**
+ * @brief Test for operation (RPC or Action) statements
+ */
+#define LY_STMT_IS_OP(STMT) (((STMT) == LY_STMT_ACTION) || ((STMT) == LY_STMT_RPC))
+
+/**
+ * @brief Test for schema node (anydata, anyxml, choice, container, leaf, leaf-list and list) statements.
```

Comment:
I am missing `case` here and even after seeing the comment with it just below the reason is not much clearer. If this is supposed to be used only for top-level nodes, it should be mentioned. But then why there is no `uses`, for example?

Reply:
It's because the current use of this macro, which comes from yang-data needs (no `uses` because it is limited to compiled statements, no `case` since yang-data does not allow it and it is not clear how to actually handle cases in top level of extension). You are right that it is supposed to be more generic so it should be reflected. I'll try to improve the macro as well as its description.